plasmapy/diagnostics/charged_particle_radiography.py

```diff
@@ -1088,6 +1104,55 @@ def run(
             f"{self.fract_deflected*100}%"
         )
 
+        return self.output
+
+    def _store_result(self):
```

Comment:
Even if a function/method is private, it still needs a docstring.  In this case the docstring is written for the developer's knowledge and not the end user. ... Always keep in the back of you mind, "Will this be understood by me or anyone else a year from now?"  If no, then the documentation/comments need to be expanded.